samples/vsphere/appliances/local_accounts.py

```
@@ -0,0 +1,122 @@
+* NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+"""
+
+from getpass import getpass
+from pprint import pprint
+
+from vmware.vapi.vsphere.client import create_vsphere_client
+from com.vmware.vapi.std.errors_client import NotFound, InvalidArgument, AlreadyExists
+
+from samples.vsphere.common import sample_cli
+from samples.vsphere.common import sample_util
+from samples.vsphere.common.ssl_helper import get_unverified_session
+
+
+class LocalAccounts:
+
+    def __init__(self):
+        parser = sample_cli.build_arg_parser()
+        args = sample_util.process_cli_args(parser.parse_args())
+        session = get_unverified_session() if args.skipverification else None
+        self.client = create_vsphere_client(server=args.server,
+                                            username=args.username,
+                                            password=args.password,
+                                            session=session)
+        self.local_accounts = self.client.appliance.LocalAccounts
+
+    def run(self):
+        """
+        Running the workflow for Local accounts.
+        It creates local account and updates its default security settings.
+        Deletes the created local account at the end.
+        """
+
+        print("Listing available accounts")
+        self.list_accounts()
+
+        print("Create local account for yourself")
+        local_user = self.create_local_account()
+
+        print("Updating the local accounts security settings")
+        self.update_local_account_security(local_user)
+
+        print("Get information for specific account")
+        self.get_account_info()
+
+        print("Deleting the local account")
+        self.delete_local_account(local_user)
+
+    def get_account_info(self):
+        try:
+            username = input("username ::")
+            pprint(self.local_accounts.get(username))
+        except NotFound as e:
+            print("Local Account mentioned is not found")
```

You can raise an ValueError here, instead of just print.